src/components/search/search.style.js

```diff
@@ -124,6 +124,7 @@ const StyledSearch = styled.div`
       }
 
       ${StyledFormField} {
+        margin-bottom: 0px;
```

Comment:
This comment still stands.
https://github.com/Sage/carbon/pull/4281#discussion_r679911373